docs/source/examples/relaxation.py

```diff
@@ -0,0 +1,38 @@
+"Relaxation examples"
+
+from gpkit import Variable, Model
+x = Variable("x")
+x_min = Variable("x_min", 2)
+x_max = Variable("x_max", 1)
+m = Model(x, [x <= x_max, x >= x_min])
+print "Original model"
+print "=============="
+print m
+print
+# m.solve()  # raises a RuntimeWarning!
+
+print "With constraints relaxed equally"
+print "================================"
+from gpkit.constraints.relax import RelaxAll
+relaxall = RelaxAll(m)
```

Comment:
After talking yesterday, I think we're agreed that the use case you're thinking of is for now better done manually?